packages/website/ts/components/fill_order.tsx

```diff
@@ -530,18 +526,18 @@ export class FillOrder extends React.Component<FillOrderProps, FillOrderState> {
         }
 
         const signedOrder = this.props.blockchain.portalOrderToSignedOrder(
```

Comment:
Lol, we can get rid of this method and simply assign the signedOrder (with bigNumbers converted).